docs/guides/buckets.mdx

```diff
@@ -0,0 +1,338 @@
+---
+sidebar_position: 3
+title: Buckets
+description: Complete guide to Buckets in ReductStore
+---
+
+<head>
+  <link rel="canonical" href="https://www.reduct.store/docs/guides/buckets"/>
+</head>
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Buckets
+
+## Concepts
+
+A bucket is a container used for data storage and serves as a logical grouping of data. It stores time-series data, with each entry having a unique name. If you're familiar with databases, think of a bucket as a database, and entries as tables.
+
+Given the nature of time-series data, ReductStore partitions data into blocks. Each block, stored in a separate file, contains a set of records. There are limits to the block size and the number of records it can hold. When a block reaches its limit, a new block is created to store additional data. This method allows ReductStore to store and access data efficiently.
+You can read more about blocks and records in the [How Does It Work?](../how-does-it-work) document.
+
+Unlike other blob storage or file systems, ReductStore has a flat hierarchy. It doesn't use folders or directories. Instead, all data is stored in entries, which are grouped by buckets.
+
+
+## Bucket Settings
+
+Each bucket has settings that determine how data is stored and accessed. These settings include:
+
+- **Quota Type**: This defines the quota type for the bucket. You can set the quota to either NONE or FIFO, with the default being NONE.
+- **Quota Size**: This represents the quota size in bytes. The default is 0, which indicates no quota.
+- **Max Block Size**: This is the maximum allowable size of a block, in bytes. The default size is 64MB.
+- **Maximal Number of Records**: This is the maximum number of records that can be contained in a block. The default number is 256.
```

Comment:
Is it correct to say the sentence below ? 

The actual number of records per block varies with record size. Setting a moderate limit is essential for fast data retrieval and efficient querying.

Reply:
I'm explain it in `### Maximal Number of Records`